coda_docs.gemspec

```diff
@@ -34,11 +34,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.16'
+  spec.add_development_dependency 'bundler'#, '~> 1.16'
```

Comment:
is there a reason why we fixed bundler to a specific version?

Reply:
No there isnt't